Source/Core/Core/HW/GCMemcard/GCMemcard.cpp

```diff
@@ -94,8 +95,14 @@ GCMemcard::GCMemcard(const std::string& filename, bool forceCreation, bool shift
   }
   if (m_sizeMb != BE16(hdr.SizeMb))
   {
-    PanicAlertT("Memory card file size does not match the header size");
-    return;
+    // Special case: Datel's "MaxDrive" cards report a size of 0x40,
+    // which should be 8,192*1,024 bytes, but the actual size of an
+    // image of such a card is 8,192*1,000. The cards work, though.
+    if ((m_sizeMb != MemCard995Mb) || (BE16(hdr.SizeMb) != 0x40))
```

Comment:
Yes, I am aware. My preference is to put the parens in for clarity, because it's one less thing to remember.

I probably should have used MemCard1019Mb instead of 0x40.

Reply:
Right, but the general convention in the codebase is to not use them when they aren't necessary like this for trivial conditions (which is why I brought it up).